### DISCUSSION TITLE
Doubt regarding Kube builder

### DISCUSSION THREAD

**Discussion by rsurati** [kubernetes-sigs/kubebuilder#3941] [Support] [answered]
I want to build basic Kubernetes controller.

https://book.kubebuilder.io/

Why this documentation is not accessible?

**Comment by camilamacedo86** [comment, marked as answer]
We apologize for the inconvenience.

Our infrastructure team is currently working on transferring the domain to facilitate better management. This process is expected to be completed within the next 24 hours.

For more details, please refer to the Slack discussion: [Link to Slack Thread](https://kubernetes.slack.com/archives/CCK68P2Q2/p1716244720415899?thread_ts=1716187409.029509&cid=CCK68P2Q2).

In the meantime, you can access the documentation from the preview of PRs in the master branch here: [Preview Documentation](https://deploy-preview-3943--kubebuilder.netlify.app/).

Thank you for your patience and understanding.

**Comment by camilamacedo86** [comment]
Solved
